src/Microsoft.CodeAnalysis.Razor.Workspaces/AcceptedCharacters.cs

```diff
@@ -3,7 +3,7 @@
 
 using System;
 
-namespace Microsoft.VisualStudio.LanguageServices.Razor
+namespace Microsoft.CodeAnalysis.Razor
```

Comment:
> iterate from there

I don't want to do that. I want to move things to the appropriate places, and clean them up (if needed) in one step